frodo.js

```diff
@@ -3,34 +3,33 @@
  *
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.